.gitignore

```diff
@@ -5,6 +5,9 @@
 !__init__.py
 !__main__.py
 
+# don't skip GitHub Actions files and directories
+!.github/**
```

Comment:
Why would git skip `.github` folder if there were no this line?

Reply:
Git skips `.github/build_app` directory because of `[Bb]uild*/` line in gitignore.